### DISCUSSION TITLE
How can I find the XML file corresponding to the monster AI?

### DISCUSSION THREAD

**Discussion by yevying** [AlanMorel/MapleServer2#830] [General]
How can I find the XML file corresponding to the monster AI?

**Comment by AngeloTadeucci** [comment]
Monster AI is not available on the client

**Reply by yevying** [reply]
Where are the monster's skill and movement data stored? The npc-metadata only contains a small amount of animation and skill data, which seems incomplete.

**Comment by yevying** [comment]
Where are the monster's skill and movement data stored? The npc-metadata only contains a small amount of animation and skill data, which seems incomplete.

**Comment by AngeloTadeucci** [comment]
> How can I find the XML file corresponding to the monster AI?

https://forum.ragezone.com/threads/maplestory2-server-side-xml-files.1228334/
